library/common/jni/jni_utility.cc

```diff
@@ -20,7 +20,10 @@ JavaVM* get_vm() { return static_jvm; }
 
 void set_class_loader(jobject class_loader) { static_class_loader = class_loader; }
 
-jobject get_class_loader() { return static_class_loader; }
+jobject get_class_loader() {
+  ASSERT(static_class_loader);
```

Comment:
I wonder whether this should be a `RELEASE_ASSERT` - it should basically always crash, even in prod. That way we could find out about potential issues as soon as possible. What do you think?

Reply:
done